src/idlcxx/src/streamer_generator.c

```diff
@@ -51,6 +51,9 @@ format_ostream_indented(indent ? ctx->depth*2 : 0, ctx->key_write_stream, __VA_A
 #define format_key_read_stream(indent,ctx, ...) \
```

Comment:
I haven't actually looked at this file: commenting on the expected test ouput below was an easier.

Reply:
@eboasson Thanks for all improvements and suggestions, I have taken large parts of your code and put it into an upcoming PR which has numerous other improvements.

I am closing this PR as it has been overtaken by other developments.